pkg/trace/customtagsRC/custom_tags_rc_handler.go

```diff
@@ -0,0 +1,87 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
You can delete this file at this point, we're not going to use the customtagsRC handler.